packages/babel-parser/src/tokenizer/index.js

```diff
@@ -247,17 +283,16 @@ export default class Tokenizer extends ParserErrors {
 
   nextToken(): void {
     const curContext = this.curContext();
-    if (!curContext?.preserveSpace) this.skipSpace();
+    if (!curContext.preserveSpace) this.skipSpace();
```

Comment:
The default value of `context` is `[ct.braceStatement]` so `curContext` is always non-nullish.